Makefile

```
@@ -144,7 +150,7 @@ tests-privileged:
 	done
 	$(MAKE) generate-cov
 
-start-kvstores:
+start-kvstores: ## Start running KVStores required for running Cilium unit-tests. More specifically this will run etcd and consul containers.
```

Nit: for consistency
```suggestion
start-kvstores: ## Start running kvstores required for running Cilium unit-tests. More specifically this will run etcd and consul containers.
```